### PR TITLE
Add docker dash support

### PR DESCRIPTION
**Pull request by JShep-tri**
- Adds support for a two-liner for getting a local `piro` server up and running
 - Rework of the `README` - would like to hear thoughts on this, I tried to segment things out in an intuitive way but it still seems a bit patchy to me

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ RUN python3.7 -m pip install --upgrade \
 
 WORKDIR /home/$USERNAME/piro/
 
-# Set ip and 
-RUN find . -name 'app.py' | xargs sed -i "s/app.run_server(debug=True)/app.run_server(host='0.0.0.0', port=8888, debug=True)/g"
+# Set ip and port of Dash app for Docker
+RUN find . -name 'app.py' | xargs sed -i "s/app.run_server(debug=True)/app.run_server(host='0.0.0.0', port=8080, debug=True)/g"
```
Comment (ChrisFajardo-TRI) [inline Dockerfile:40]: Just noting that we should eventually move host/port to a config file

**Comment by MuratAykol-TRI** [comment]
@JShep-tri the edits to README look good to me.

File: Dockerfile [hunk 2]
```diff
@@ -36,7 +36,7 @@ RUN python3.7 -m pip install --upgrade \
...
 
 RUN python3.7 setup.py develop
```
Comment (ChrisFajardo-TRI) [inline Dockerfile:42]: putting this line at the end of the Dockerfile should work too I think, if you didn't want to add docker-compose

`ENTRYPOINT python3.7 web/app.py`

File: Dockerfile (hunk 1 above)
Comment (JosephMontoya-TRI) [reply Dockerfile:40]: Agreed, also I think we can just change app.py to have these params as defaults.

**Review by JosephMontoya-TRI — APPROVED** [review]
I'd change the app.py defaults, but otherwise looks great.

File: Dockerfile (hunk 2 above)
Comment (JShep-tri) [reply Dockerfile:42]: I wanted to keep a separate `docker run` command for those who wish to develop on `piro` within the docker container whereas `docker-compose` can be used by those who just want to spin up a local server quickly and run computations.

**Pull request merged**